lib/runner/CLI.js

```diff
@@ -40,13 +40,22 @@ class CLI {
             jestConfig.testMatch = undefined;
             jestConfig.testRegex = argv[2];
         }
-        let runInBand = false;
+
+
         const type = Configuration.instance().value("type");
         const invoker = Configuration.instance().value("invoker");
-        if (type === CONSTANTS.TYPE.e2e || invoker === CONSTANTS.INVOKER.virtualDeviceInvoker){
-            jestConfig.collectCoverage = false;
+
+        const isRunningE2E = type === CONSTANTS.TYPE.e2e || invoker === CONSTANTS.INVOKER.virtualDeviceInvoker;
+        let runInBand = Configuration.instance().value("runInBand");
```

Comment:
you can set a default value for runInBand,

Reply:
Calling `Configuration.instance().value("runInBand", undefined, true)` will allow you to skip the `typeof runInBand === "undefined"` condition. Is that what you mean @ecruzado?

Reply:
yes, it would be more readable I think